index.html

```diff
@@ -498,10 +498,10 @@ <h2>
         The meetings themselves are not open to public participation, however.
         </p>
         <p>
-          Information about the group (including details about deliverables, issues, actions, status, participants, and meetings) will be available from the <a href="https://www.w3.org/AudioVideo/TT/">Timed Text Working Group home page</a>.
+          Information about the group (including details about deliverables, issues, actions, status, participants, and meetings) will be available from the <a href="https://www.w3.org/groups/wg/timed-text">Timed Text Working Group home page</a>.
```

Comment:
I think this one should be reverted. The Homepage link should match what "Homepage" says at https://www.w3.org/groups/wg/timed-text .

Reply:
-1 
Rationale: we're trying to use as much as possible links to /groups/wg/ or /groups/ig/ in charters so that people looking at several charters and visiting the group's pages find that we expose exactly the same information from group to group in predictable places. Such pages in some cases (like TT) point to  a different homepage, so your page will be one click away.

Reply:
I don't understand: this is inconsistent. How can it be that there is a link from groups/wg/timed-text to a "Homepage" and this here link to "... home page" does not point to the same place?

Also, the sentence is clearly about where to get more details about the working mode of the WG, and that's not defined by the groups/wg/timed-text link.

> Such pages in some cases (like TT) point to a different homepage, so your page will be one click away.

This is the wrong way around. _Your_ page will be one click away! I'm happy to accept the direct links in the two other places.

Reply:
Sorry, @nigelmegitt I stand down. Reverting in line 501 does make sense indeed.

Reply:
Thank you for comments and discussions. I've totally missed that this link text contains 'homepage'. 
Let me revert this line with fix as above.

Reply:
@nigelmegitt @koalie merged. (please check again)